Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Feedback.hs

```diff
@@ -92,16 +109,25 @@ feedback request personId = do
   booking <- QRB.findById ride.bookingId >>= fromMaybeM (BookingNotFound ride.bookingId.getId)
   merchant <- CQM.findById booking.merchantId >>= fromMaybeM (MerchantNotFound booking.merchantId.getId)
   bppBookingId <- booking.bppBookingId & fromMaybeM (BookingFieldNotPresent "bppBookingId")
+  (mediaId, filePath) <- case request.mbAudio of
+    Just audio -> do
+      let fileType = S3.Audio
+          contentType = T.pack "mp3"
+      filePath <- S3.createFilePath "/feedback-media/" ("bppBookingId-" <> bppBookingId.getId) fileType contentType
+      media <- audioFeedbackUpload (personId, merchant.id) audio filePath fileType
+      return (Just media.fileId, Just filePath)
+    Nothing -> do
+      return (Nothing, Nothing)
   issueId' <- getIssueIdForRide booking
   _ <- QPFS.updateStatus booking.riderId DPFS.IDLE
   _ <- QRide.updateRideRating rideId ratingValue
   ratingu <- QRating.findRatingForRide rideId
   _ <- case ratingu of
     Nothing -> do
-      newRating <- DRating.buildRating rideId booking.riderId ratingValue feedbackDetails request.wasOfferedAssistance
+      newRating <- DRating.buildRating rideId booking.riderId ratingValue feedbackDetails request.wasOfferedAssistance mediaId
```

Comment:
you don'e have to do anythin here...here update and create itself is wrong 
you are using this in rider call for rating..and creating the entry at the rider side itself